cmd/root.go

```diff
@@ -76,39 +79,65 @@ func New(o *options.Options) *cobra.Command {
 	return cmd
 }
 
-// rootCmd runs scorecard checks given a set of arguments.
-func rootCmd(o *options.Options) error {
-	var err error
-	var repoResult scorecard.Result
+// Build the list of repositories to scan, honoring --repos > --org > --local > --repo/pkg-managers.
+func buildRepoURLs(ctx context.Context, o *options.Options) ([]string, error) {
+	// --repos has highest precedence
+	if len(o.Repos) > 0 {
+		var urls []string
+		for _, r := range o.Repos {
+			r = strings.TrimSpace(r)
+			if r != "" {
+				urls = append(urls, r)
+			}
+		}
+		return urls, nil
+	}
+
+	// --org: expand to all non-archived repos
+	if o.Org != "" {
+		// create a transport to respect auth, rate limiting, etc.
+		logger := sclog.NewLogger(sclog.DefaultLevel)
+		rt := roundtripper.NewTransport(ctx, logger)
+		repos, err := orgpkg.ListOrgRepos(ctx, o.Org, rt)
```

Comment:
you wrote `ListOrgRepos` to do all this for you if `rt == nil` so might as well just do:
```
repos, err := orgpkg.ListOrgRepos(ctx, o.Org, nil)
```

Reply:
Damn 🤦 This was the previous behavior.  Changed.